region_test.go

```diff
@@ -100,3 +100,43 @@ func TestGetRegion(t *testing.T) {
 		})
 	}
 }
+
+func TestVPCRegionForPowerVSRegion(t *testing.T) {
```

Comment:
good to see some tests added but I don't see any tests running in the job because we don't have any ut enabled for this project.. Wondering if we can enable something in GH action to run these tests(in a different PR)

Reply:
I can take that up, may need a little help too.

Reply:
cool, now I see tests are running with the PR.. Lets review and merge the code.